commands/userargs.go

```diff
@@ -19,6 +21,10 @@ func getUsersFromUserArgs(c client.Client, userArgs []string) []*model.User {
 }
 
 func getUserFromUserArg(c client.Client, userArg string) *model.User {
+	if checkTraversal(userArg) {
```

Comment:
I think we support usernames with `..`, so we should check `..` on the emails and `/` on the usernames in this case